src/types/ApiError.ts

```diff
@@ -1,3 +1,15 @@
+export enum ErrorCode {
+    GenericError = "generic_error",
```

Comment:
Is GenericError or InvalidArgument used?

Reply:
I don't think so. I just added them for completeness